config/routes.rb

```diff
@@ -3,7 +3,10 @@
 Rails.application.routes.draw do
   root 'users#index'
   resources :users, only: [:index, :show] do
-  resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new, :create] do
+      resources :comments, only: [:new, :create]
+      resources :likes, only: [:new, :create]
+    end
```

Comment:
✅ There are routes in the `config/routes.rb` file for creating posts 👍🏻 
✅ There are routes in the config/routes.rb file for creating comments.